src/components/dashboard/sidebar/SubsidiesSummary.jsx

```diff
@@ -162,16 +167,21 @@ const SubsidiesSummary = ({
           <LearnerCreditSummaryCard
             className="border-0 shadow-none"
             expirationDate={learnerCreditSummaryCardData.expirationDate}
+            assignmentOnlyLearner={assignmentOnlyLearner}
           />
         )}
       </div>
-      {(searchCoursesCta && !assignmentOnlyLearner) && (
-        <SidebarCard
-          cardClassNames="border-0 shadow-none"
-        >
-          {searchCoursesCta}
-        </SidebarCard>
-      )}
+      {(searchCoursesCta && !assignmentOnlyLearner)
+        ? (
+          <SidebarCard cardClassNames="border-0 shadow-none">
+            {searchCoursesCta}
+          </SidebarCard>
+        )
+        : (
+          <SidebarCard cardClassNames="border-0 shadow-none">
+            {contactAdministratorCTA}
+          </SidebarCard>
+        )}
```

Comment:
[clarification, sanity-check] Are there any other subsidies than Learner Credit where this `contactAdministratorCTA` would appear (given `SubsidiesSummary` works with all subsidies, not just Learner Credit).

Reply:
From the looks of groking all the subsidy summary cards individually, there doesnt appear to be another instance where contact administrator is used in the side bar card specifically. But I will leave a node before the conditional that renders the contactAdminCTA and the component that this can be generalized for future use 👍🏽

Reply:
@brobro10000 I just wanted to check in here again because I believe the new "Contact admin" CTA _will_ appear for non-LC subsidy types (e.g., subscriptions), given certain circumstances are met. The conditional will indeed fallback to the "Contact admin" CTA when there is assignments only; however it'll _also_ fallback to the "Contact admin" CTA if the other conditions around whether `searchCoursesCta` is defined.

`searchCoursesCta` is _not_ defined in the following situations (where it'd fallback to the "Contact admin" CTA currently):
* `SubsidiesSummary` renders on the program progress page (i.e., CTA should only be visible on dashboard page route).
* Enterprise customer has Learner Portal search _disabled_
* `showSearchCoursesCta` prop is falsey (though it looks like this prop is always `true` today).

For example, a learner with only a subscription license who's customer has disabled Learner Portal search, the "Contact admin" CTA will still appear:

<img width="392" alt="image" src="https://github.com/openedx/frontend-app-learner-portal-enterprise/assets/2828721/16a084ed-5915-43f2-958e-239678b243d3">

FWIW, the [ticket](https://2u-internal.atlassian.net/browse/ENT-8167) doesn't really denote any requirements around changing behavior of other subsidy types, hence the question/concern. Really, the ticket doesn't mention implementing a "Contact admin" CTA button either (my read of the ticket mentions changing the text only, without a CTA). I might also wonder whether having this "Contact admin" CTA is a bit redundant, given there is already a "Contact admin" CTA in the "Need help?" box below the subsidies summary box (as seen above).